### DISCUSSION TITLE
How to hide console using pyinstaller .exe?

### DISCUSSION THREAD

**Discussion by AlexPaiva** [ultrafunkamsterdam/undetected-chromedriver#1066] [Q&A]
So, I have a very simple setup:
`driver = uc.Chrome(use_subprocess=True)`

If I run `python myApp.py` on the terminal, the browser opens and no console window.

If I build a .exe using pyinstaller, now a console window opens up when the browser opens up.

Thing is, I am using all the correct settings in the .spec file to generate no console by using, for example: `console=False,` on the .spec file. And no console shows up, unless for UC. On regular selenium it stays hidden but for UC it shows up.

Anyone know how to hide this stubborn window?

**Comment by huyangnl** [comment]
`driver = uc.Chrome(version_main=chromeversion, options=options, use_subprocess=True, service_creationflags=CREATE_NO_WINDOW)`

it could remove and hide black console..
